src/go/k8s/api/redpanda/v1alpha2/redpanda_types.go

```diff
@@ -48,6 +48,9 @@ type RedpandaSpec struct {
 	ChartRef ChartRef `json:"chartRef,omitempty"`
 	// Defines the Helm values to use to deploy the cluster.
 	ClusterSpec *RedpandaClusterSpec `json:"clusterSpec,omitempty"`
+	// Deprecated: Old Cluster and Console would not be migrated to flux based version.
```

Comment:
it's a bit more than deprecated 😅  To further reduce confusion, I think it'd be helpful to fully replace the description with something like "Deprecated and Removed in <Next version>. Downgrade to <current version> to perform the migration"